frontend/src/scenes/notebooks/Nodes/NodeWrapper.tsx

```diff
@@ -165,9 +166,9 @@ export function NodeWrapper<T extends CustomNotebookNodeAttributes>({
                                         />
                                     )}
 
-                                    {!!widgets.length && isEditable ? (
+                                    {widgets.length > 0 ? (
                                         <LemonButton
-                                            onClick={() => setWidgetsVisible(true)}
+                                            onClick={() => setIsShowingSidebar(true)}
```

Comment:
I feel like this should toggle, rather than just showing. And it would be great if it could have an "active" state if open.